### PR TITLE
Go 1.18 compatibility #patch

### PR DESCRIPTION
**Pull request by extemporalgenome**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+GO_CI_VERSION = v1.46.1
 BINARIES = aws-env
 WD ?= $(shell pwd)
 NAMESPACE=sendgrid
 APPNAME=aws-env
 
 export GIT_COMMIT ?= $(shell git rev-parse --verify HEAD)
 export BUILD_DATE ?= $(shell date -u)
-export VERSION ?= $(or $(shell cat version_wf),0.0.1)
+export VER_VERSION ?= 0.0.1
```
Comment (mlueckest) [inline Makefile:10]: Does this warrant at least a patch bump or do we generally not increment the shell script?

File: Makefile [hunk 2]
```diff
@@ -39,8 +39,11 @@ push:
 
 .PHONY: push-tagged
 push-tagged:
-	docker tag docker.sendgrid.net/$(NAMESPACE)/$(APPNAME) docker.sendgrid.net/$(NAMESPACE)/$(APPNAME):$(VERSION)
-	docker push docker.sendgrid.net/$(NAMESPACE)/$(APPNAME):$(VERSION) 
+	docker tag \
+		"docker.sendgrid.net/$(NAMESPACE)/$(APPNAME)"
```
Comment (mlueckest) [inline Makefile:43]: Should this be ending in `\` ?

**Review by mlueckest — CHANGES_REQUESTED** [review]
Just a couple of minor comments.

File: Makefile (hunk 2 above)
Comment (extemporalgenome) [reply Makefile:43]: Indeed

File: Makefile [hunk 3]
```diff
@@ -39,8 +39,11 @@ push:
...
+		"docker.sendgrid.net/$(NAMESPACE)/$(APPNAME):$(VER_DOCKER_TAG)"
```
Comment (extemporalgenome) [inline Makefile:44]:
```suggestion
	docker tag \
		"docker.sendgrid.net/$(NAMESPACE)/$(APPNAME)" \
		"docker.sendgrid.net/$(NAMESPACE)/$(APPNAME):$(VER_DOCKER_TAG)"
```

File: Makefile (hunk 1 above)
Comment (mlueckest) [reply Makefile:10]: Got it, thank you, this clarifies the use of VER_VERSION for me.

**Review by mlueckest — APPROVED** [review]

**Pull request merged**